service_linux.go

```diff
@@ -102,13 +102,11 @@ func (n *network) addLBBackend(ip net.IP, lb *loadBalancer) {
 	}
 	ep, sb, err := n.findLBEndpointSandbox()
 	if err != nil {
-		logrus.Errorf("error in addLBBackend for %s/%s for %v", n.ID(), n.Name(), err)
+		logrus.Errorf("addLBBackend %s/%s: %v", n.ID(), n.Name(), err)
 		return
 	}
 	if sb.osSbox == nil {
-		return
-	}
-	if n.ingress && !sb.ingress {
+		logrus.Errorf("addLBBackend %s/%s: unable to find LB OS sandbox", n.ID(), n.Name())
```

Comment:
In the past I remember these 2 to be very frequent, is it still the case? if this is not a fatal condition I would not use Error.